src/test/java/com/google/api/codegen/gapic/testdata/php/php_library.baseline

```diff
@@ -1505,7 +1505,7 @@ function samplePublishSeries()
     $seriesUuid = new SeriesUuid();
 
     try {
-        $response = $libraryServiceClient->publishSeries($shelf, $books, $seriesUuid, ['edition' => $edition]);
+        $response = $libraryServiceClient->publishSeries($shelf, $books, $seriesUuid);
```

Comment:
What's the significance of this changing? Was it not right before?

Reply:
same as above

Reply:
Right. This is also due to that we are not using the correct `initCode`. See https://github.com/googleapis/gapic-generator/pull/2883/files#diff-70645adb9110db92a78ae7061153d8ddL168.

(Also we don't have `edition` at all :p)